src/sentry/api/endpoints/issue_occurrence.py

```diff
@@ -125,18 +124,17 @@ def post(self, request: Request) -> Response:
                 status=status.HTTP_400_BAD_REQUEST,
             )
 
+        event.update({"data": event.pop("data", {})})
+
         event_serializer = BasicEventSerializer(data=event)
         if not event_serializer.is_valid():
             return Response(event_serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
         event = event_serializer.validated_data
 
-        # hack to put in unserialized data
-        unserialized_data = event.pop("data")
-        if unserialized_data:
-            event.update(unserialized_data)
-
         occurrence["event_id"] = str(event["event_id"])
+        # XXX: not sure if evidence_data is
+        occurrence.update({"evidence_data": occurrence.pop("evidence_data", {})})
```

Comment:
same comment here

Reply:
Wait, maybe I'm misunderstanding this one. Isn't `evidence_data` nested within `occurrence` and further down we unpack `occurrence`?

Reply:
oh right. is this because it's not present in the sample payload for profiling? uh hmm use `setdefault` then https://www.w3schools.com/python/ref_dictionary_setdefault.asp

Reply:
we'll make a decision about whether it's required later